components/new-contribution-flow/StepPayment.js

```diff
@@ -1,16 +1,283 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
+import { useQuery } from '@apollo/react-hooks';
+import themeGet from '@styled-system/theme-get';
+import { find, first, get, isEmpty, uniqBy } from 'lodash';
+import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
 
-import { Box } from '../../components/Grid';
+import { CollectiveType } from '../../lib/constants/collectives';
+import { API_V2_CONTEXT, gqlV2 } from '../../lib/graphql/helpers';
+import { getPaymentMethodName } from '../../lib/payment_method_label';
+import { getPaymentMethodIcon, getPaymentMethodMetadata } from '../../lib/payment-method-utils';
 
-class NewContributionFlowStepPayment extends React.Component {
-  static propTypes = {
-    collective: PropTypes.object,
-  };
+import Container from '../../components/Container';
+import { Box, Flex } from '../../components/Grid';
+import CreditCardInactive from '../../components/icons/CreditCardInactive';
+import Loading from '../../components/Loading';
+import MessageBox from '../../components/MessageBox';
+import NewCreditCardForm from '../../components/NewCreditCardForm';
+import { withStripeLoader } from '../../components/StripeProvider';
+import StyledButton from '../../components/StyledButton';
+import StyledHr from '../../components/StyledHr';
+import StyledRadioList from '../../components/StyledRadioList';
+import { P } from '../../components/Text';
+import { withUser } from '../../components/UserProvider';
+
+const PaymentMethodBox = styled(Container)`
+  border-top: 1px solid ${themeGet('colors.black.300')};
+  display: flex;
+  flex-direction: column;
+  background: ${themeGet('colors.white.full')};
+  &:hover {
+    background: ${themeGet('colors.black.50')};
+  }
+`;
 
-  render() {
-    return <Box width={1}>Payment method chooser - same for all states</Box>;
+const paymentMethodsQuery = gqlV2/* GraphQL */ `
+  query UpdatePaymentMethodPopUpPaymentMethod($slug: String) {
```

Comment:
Update the query name so that it's unique within the codebase and consistent with where it is. Ie: `ContributionFlowPaymentMethods`